docs/api/networksocket/CellularNonIPSocket.md

```diff
@@ -0,0 +1,88 @@
+## Non-IP cellular socket
+
+The CellularNonIPSocket class provides, through standard socket `send` and `recv` member functions, the ability to send and receive 3GPP non-IP datagrams (NIDD) using our cellular IoT feature. This feature is implemented in the [`ControlPlane_netif`](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_control_plane__netif.html) class.
+
+The constructor takes no parameters. To initialize the socket on a specified NetworkInterface, you must call the `open` method, which takes a CellularContext pointer.
+
+[`CellularContext`](https://os.mbed.com/docs/development/mbed-os-api-doxy/_cellular_context_8h.html) sets up the modem into the Control Plane optimization mode of operation if it is requested and if the cellular network supports it.
+
+Control plane optimization is a new feature for cellular IoT (CIoT). With it, the device can use cellular control channels for data communications, to save power and resources. It does this by using less radio singling.
```

Comment:
Is "radio singling" correct, or is it "radio signaling"?

Reply:
signaling is the correct term

Reply:
About the rephrasing. Is it clear that "this" in "It does **this** by using" refers to saving power and resources?

Reply:
No, it's not clear. Good catch. What do you think about "...to save power and resources by using less radio signaling."?

Reply:
Sounds good!